### ISSUE TITLE
Relax the dependecies

### ISSUE THREAD

**Issue by edravis** [BuiltByBig/react-fontawesome#2]
So, I'm trying to install it on Reactjs 0.13.2, but:

```
the package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-font-awesome@1.0.3 wants react@~0.11.2
npm ERR! peerinvalid Peer react-fontawesome@0.1.0 wants react@0.13.x
```

**Comment by danawoodman** [comment]
Dropped it down to `0.12`+, let me know if that works for you. Thanks for the PR!

**Comment by danawoodman** [comment]
Fixed in version `0.2.5` published to npm.

**Comment by edravis** [comment]
Thanks dude, I'll try again. I solved by bundle my own Fontawesome package ;)
